api_tests/lib_sdk/cnd_instance.ts

```diff
@@ -56,6 +60,12 @@ export class CndInstance {
             ],
         });
 
+        if (global.verbose) {
```

Comment:
Should we have a function helper that does the conditional check and log passed message?

Reply:
Happy to have that but not in this PR. I just pushed it because the work was already done as part of debugging :)